olive/passes/onnx/optimum_conversion.py

```diff
@@ -66,46 +84,66 @@ def validate_search_point(
     def _run_for_config(
         self, model: PyTorchModelHandler, data_root: str, config: Dict[str, Any], output_model_path: str
     ) -> Union[ONNXModelHandler, CompositeModelHandler]:
+        if not model.hf_config:
```

Comment:
>hf_config [](http://example.com/codeflow?start=21&length=9)

It seems the hf_config.task or hf_config.model_class is not needed by optimum conversion. 

[https://github.com/microsoft/Olive/blob/3037a4f8cf51f6a559279f8bd20a81a5bcae0f98/examples/mistral/mistral_optimize.json#L7](https://github.com/microsoft/Olive/blob/3037a4f8cf51f6a559279f8bd20a81a5bcae0f98/examples/mistral/mistral_optimize.json#L7)

So, are there any better way to make them optional?

Reply:
It is required if we need to load the input pytorch model and evaluate it. So I think, it is better to keep it.